apps/expo/src/components/ui/button/button.tsx

```diff
@@ -63,7 +62,7 @@ const Button = ({
     <Pressable
       {...props}
       className={cn(buttonVariants({ color, size, rounded }), className)}
-      testID={testID}
+      testID={props.testID}
```

Comment:
This line can also be removed

Reply:
I need it for the test file

Reply:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/0169adc9-fc63-4811-9482-c709bfd3d18b)